src/index.js

```diff
@@ -3,38 +3,82 @@ import renderPage from './modules/renderPg.js';
 
 renderPage();
 
-const scores = [
-  {
-    user: 'John',
-    score: 50,
-  },
-  {
-    user: 'Jane',
-    score: 100,
-  },
-  {
-    user: 'Jack',
-    score: 150,
-  },
-];
-
-scores.forEach(() => {
-  const scoresTable = document.getElementById('score-table');
-  scoresTable.innerHTML = '';
-  const table = document.createElement('table');
-  table.classList.add('table');
-
-  const tbody = document.createElement('tbody');
-
-  scores.forEach((item, index) => {
-    const row = document.createElement('tr');
-    row.classList.add(index % 2 === 0 ? 'even' : 'odd');
-    const cell = document.createElement('td');
-    cell.textContent = `${item.user}: ${item.score}`;
-    row.appendChild(cell);
-    tbody.appendChild(row);
-  });
-
-  table.appendChild(tbody);
-  scoresTable.appendChild(table);
+document.addEventListener('DOMContentLoaded', () => {
+  // eslint-disable-next-line no-unused-vars
+  const gameID = 'LSRlW6OdY2zWcbqajLTY';
+  const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+  const requestURL = `${baseURL}games/${gameID}/scores/`;
+
+  const form = document.querySelector('form');
+
+  // POST scores to the API
+  const createScore = async (inputName, inputValue) => {
+    const response = await fetch(requestURL, {
+      method: 'POST',
+      body: JSON.stringify({
+        user: inputName,
+        score: inputValue,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+    // eslint-disable-next-line no-unused-vars
```

Comment:
## [OPTIONAL]

- [ ]  Kindly make sure any linter checker is not disabled, they are really important as the help you structure your code and prevent future bugs by following  best practices, kindly solve them instead